Policies/SQLServer/Azure_SQLDatabase_Audit_Enable_Threat_Detection_Server/Readme.md

```diff
@@ -134,6 +134,17 @@ Following policy will configure/Audit security contacts (email) at Subscription
 
 ___ 
 
+#### Policy Assessment Evaluation
+
+To collect the complete audit evaluation for a control, please run the policies in below order.
+
+The assessment result should be combination of:
+
+1. [Security - SQL Server - AuditSqlServerAuditingSettings](Security%20-%20SQL%20Server%20-%20AuditSqlServerAuditingSettings.json)
+2. [Security - SQL Server - AuditSqlServerSecuritySettings](Security%20-%20SQL%20Server%20-%20AuditSqlServerSecuritySettings.json)
+3. [Security - SQL Server - AuditSqlServerThreatDetection](Security%20-%20SQL%20Server%20-%20AuditSqlServerThreatDetection.json) OR [Security - Subscription - UpdateSecurityContacts](Security%20-%20Subscription%20-%20UpdateSecurityContacts.json)
```

Comment:
Is it [(1) and (2) and (3a)] OR [(1) and (2) and (3b)]?

Reply:
Hi Zia, the result should be combination of 1 +2 + 3A/ 1+2+3B

Reply:
that's why I placed the 'OR' in point num 3